apps/src/javalab/JavalabView.jsx

```diff
@@ -192,13 +221,17 @@ class JavalabView extends React.Component {
   }
 }
 
+// We use the UnconnectedJavalabView to make this component testable with our version of react-redux.
```

Comment:
Also add in a note that this is a deprecated pattern. It's pretty common to copy/paste a component as a starting point for a new component. Given the fragility this pattern lends to tests, it'd be good to warn people away from this unless it meets their specific needs.

Optional: Make the details around why we're using it here more specific. This would allow someone who sees this in the future reason about whether using this pattern would be appropriate for their use case.